shared/common-adapters/image.desktop.tsx

```diff
@@ -15,18 +15,18 @@ const Image = (props: Props) => {
     },
     []
   )
-  const animatedStyle = useSpring({
```

Comment:
useSprings api has changed so the behavior is different. this is likely entirely not worth it so i'm removing it. if we need this back it'll be better to use css anyways